generic.go

```diff
@@ -437,3 +437,24 @@ func Partition[V any](slice []V, predicate func(v V) bool) ([]V, []V) {
 	}
 	return pass, reject
 }
+
+func divCeil(a, b int) int {
+	if a%b == 0 {
+		return a / b
+	}
+	return a/b + 1
+}
```

Comment:
where do I put a utility function like this one?

Reply:
Any particular reason for not using `math.Ceil`?
Other than that, we could put these utilities as exported functions inside the `internal/` dir.